Library/Formula/vegeta.rb

```diff
@@ -0,0 +1,26 @@
+require "formula"
+require "language/go"
+
+class Vegeta < Formula
+  homepage "https://github.com/tsenart/vegeta"
+  url "https://github.com/tsenart/vegeta/archive/v5.5.2.zip"
+  version "5.5.2"
+  sha1 "f7f61a9512fe09490ea1e0114bb0bbcf450c9a52"
+
+  depends_on "go" => :build
+
+  go_resource "github.com/bmizerany/perks" do
+    url "https://github.com/bmizerany/perks.git",
+      :revision => "6cb9d9d729303ee2628580d9aec5db968da3a607"
+  end
+
+  def install
+    mkdir_p buildpath/"src/github.com/tsenart/"
+    ln_s buildpath, buildpath/"src/github.com/tsenart/vegeta"
+    ENV["GOPATH"] = buildpath
+    Language::Go.stage_deps resources, buildpath/"src"
+
+    system 'go', 'build', '-o', 'vegeta'
+    bin.install "vegeta"
+  end
```

Comment:
Kind of hard to test the tool without launching a server I can HTTP test first. Is something along theses lines acceptable:

```ruby
  test do
    system "#{bin}/vegeta", "attack", "-h"
  end
```